lectures/intro_supply_demand.md

```diff
@@ -670,40 +664,37 @@ Using the class, plot the inverse demand and supply curves $i_d$ and $i_s$
 :class: dropdown
 ```
 
-```{code-cell} ipython3
-class Market:
-
-    def __init__(self,
-                 d_0=1.0,      # demand intercept
-                 d_1=0.6,      # demand slope
-                 s_0=0.1,      # supply intercept
-                 s_1=0.4):     # supply slope
+Let us make use of a [namedtuple](https://docs.python.org/3/library/collections.html#collections.namedtuple) container provided by Python to hold the parameters of the Market. 
 
-        self.d_0, self.d_1 = d_0, d_1
-        self.s_0, self.s_1 = s_0, s_1
-
-    def inverse_demand(self, q):
-        return self.d_0 - self.d_1 * q**0.6
-
-    def inverse_supply(self, q):
-        return self.s_0 + self.s_1 * q**1.8
+```{code-cell} ipython3
+Market = namedtuple('Market', ['d_0', # demand intercept
+                               'd_1', # demand slope
+                               's_0', # supply intercept
+                               's_1'] # supply slope
+                   )
 ```
 
-Let's create an instance.
+We can now define some functions that `create` and `operate` on these Market parameters. 
```

Comment:
I'm confused.  Don't we already have the code in 670 above, in line 363?

Either way, I suggest dropping the back ticks, which should be reserved for code.

Reply:
sorry @jstac I prematurely pushed. Working on the final `proof` read now :-).

Reply:
this duplication is no resolved after [5e2d75b](https://github.com/QuantEcon/lecture-python-intro/pull/372/commits/5e2d75bcb745ff5274c0d292bd80dae3e1ae8304). I had implemented the `namedtuple` in the exercise first and was propagating it to the text.